features/conference/navigationHandler.native.js

```diff
@@ -0,0 +1,19 @@
+import { ScreenRegistry } from '../base/navigation';
+import { CONFERENCE_SCREEN } from './constants';
+
+/**
+ * Handler for conference screen route.
+ *
+ * @param {Store} store - Redux store.
+ * @param {Object} action - Action object.
+ * @param {Navigator} action.navigator - Navigator instance.
+ * @returns {void}
+ */
+export function navigate(store, action) {
+    let route = ScreenRegistry.getScreenByName(CONFERENCE_SCREEN);
+
+    // TODO: currently replace method doesn't support animation, but work
+    // towards adding it is done in
+    // https://github.com/facebook/react-native/issues/1981
+    action.navigator.replace(route);
```

Comment:
Yes. At first I thought of having both Conference and WelcomePage mounted all the time and jump between two of them. But later it turned out that it's not convenient. Instead in https://github.com/jitsi/jitsi-meet-react/pull/56/commits/75b1fbf8ae1c240311d291beed94b709249d35b7 I've introduced approach when each screen-level component initializes itself when is mounted and disposes itself when is unmounted. I'd say this approach is much closer to React/Redux principles then the previous ones.